.gitignore

```diff
@@ -37,4 +37,4 @@ __pycache__
 .*.sw?
```

Comment:
Maybe a comment to explain this one, I don't think I know what this file is.

Reply:
I also don't know this file ending, but I didn't put it there, it is not part of this PR.